### PR TITLE
Add a new file creation step, and verify command - combiner

### PR DESCRIPTION
**Pull request by robbiethegeek**

### CODE REVIEW

File: problems/combiner/problem.txt [hunk 1]
```diff
@@ -1,5 +1,7 @@
-Write a module that returns a readable/writable stream using the
-`stream-combiner` module. You can use this code to start with:
+Create a new file named combiner.js, it should return a readable/writable stream using the
```
Comment (kalinchernev) [inline problems/combiner/problem.txt]: I like the added instruction for making a file. Let's keep also the info that it's a module? It's more node-specific explanation which I think is good for learners at this stage.

**Comment by robbiethegeek** [comment]
Thanks @kalinchernev feedback incorporated.

**Comment by kalinchernev** [comment]
Thanks @robbiethegeek for the quick update!

**Comment by robbiethegeek** [comment]
Can these be merged?

**Pull request merged**